src/main/fc/cli.c

```diff
@@ -2772,38 +2766,38 @@ STATIC_UNIT_TESTED void cliSet(char *cmdline)
                                 // note: no need to copy substrings for atoi()
                                 //       it stops at the first character that cannot be converted...
                                 switch (val->type & VALUE_TYPE_MASK) {
-                                    default:
-                                    case(VAR_UINT8): {
-                                        // fetch data pointer
-                                        uint8_t *data = (uint8_t *)getValuePointer(val) + i;
-                                        // store value
-                                        *data = (uint8_t)atoi((const char*) valPtr);
-                                        }
-                                        break;
-
-                                    case(VAR_INT8): {
-                                        // fetch data pointer
-                                        int8_t *data = (int8_t *)getValuePointer(val) + i;
-                                        // store value
-                                        *data = (int8_t)atoi((const char*) valPtr);
-                                        }
-                                        break;
-
-                                    case(VAR_UINT16): {
-                                        // fetch data pointer
-                                        uint16_t *data = (uint16_t *)getValuePointer(val) + i;
-                                        // store value
-                                        *data = (uint16_t)atoi((const char*) valPtr);
-                                        }
-                                        break;
-
-                                    case(VAR_INT16): {
-                                        // fetch data pointer
-                                        int16_t *data = (int16_t *)getValuePointer(val) + i;
-                                        // store value
-                                        *data = (int16_t)atoi((const char*) valPtr);
-                                        }
-                                        break;
+                                default:
+                                case VAR_UINT8: {
+                                    // fetch data pointer
+                                    uint8_t *data = (uint8_t *)getValuePointer(val) + i;
+                                    // store value
+                                    *data = (uint8_t)atoi((const char*) valPtr);
```

Comment:
I'm not against using brackets in `switch` statements, just against using them when they can be easily avoided.  So I wouldn't get rid of them here.